kubernetes/samples/scripts/create-weblogic-domain/model-in-image/README.md

```diff
@@ -452,3 +289,175 @@ the pods are up, you can ctrl-c to exit the build script.
+
+     > __NOTE__: If a local docker login and manual pull of `container-registry.oracle.com/database/enterprise:12.2.0.1-slim` is not sufficient (for example, if you are using a remote k8s cluster), then uncomment the imagePullSecrets stanza in '$WORKDIR/k8s-db-slim.yaml' and create the image pull secret:
+       ```
+       kubectl create secret docker-registry regsecret \
+         --docker-server=container-registry.oracle.com \
+         --docker-username=your.email@some.com \
+         --docker-password=your-password \
+         --docker-email=your.email@some.com 
+       ```
+
+   - Use the sample script in '$SRCDIR/kubernetes/samples/scripts/create-oracle-db-service' to create an Oracle DB running in pod 'oracle-db'.
+
+     > __NOTE__: If your database image access requires the `regsecret` image pull secret that you optionally created above, then pass `-s regsecret` to the `start-db-service.sh` command line.
+
+     ```
+     cd $SRCDIR/kubernetes/samples/scripts/create-oracle-db-service 
+     start-db-service.sh
+     ```
+
+     This script will deploy a database with URL `oracle-db.default.svc.cluster.local:1521/devpdb.k8s` and administration password `Oradoc_db1`.
+ 
+     > __WARNING__: The Oracle Database Docker images are only supported for non-production use. For more details, see My Oracle Support note: Oracle Support for Database Running on Docker (Doc ID 2216342.1) 
+     >            : All the data is gone when the database is restarted. 
+   
+     > __NOTE__: This step is based on the steps documented in [Run a Database](https://oracle.github.io/weblogic-kubernetes-operator/userguide/overview/database/).
+
+2. Use the sample script in `WORDIR/kubernetes/samples/scripts/create-rcu-schema` to create the RCU schema with schema prefix `FMW1`.
+
+   Note that this script assumes `Oradoc_db1` is the dba password, `Oradoc_db1` is the schema password, and that the database URL is `oracle-db.default.svc.cluster.local:1521/devpdb.k8s`.
+
+   ```
+   cd $SRCDIR/kubernetes/samples/scripts/create-rcu-schema 
+   ./create-rcu-schema.sh -s FMW1 -i container-registry.oracle.com/middleware/fmw-infrastructure:12.2.1.3
+   ```
+
+3. __NOTE__:  If you need to drop the repository, you can use this command in the terminal:
+
+   ```
+   drop-rcu-schema.sh -s FMW1
+   ```
+
+
+## Increase Introspection Job Timeout
+
+Since JRF domain home creation takes considerable time, and since model-in-image creates your domain home for you using the introspection job, you should increase the timeout for the introspection job. Use the command `kubectl -n sample-weblogic-operator-ns edit configmap weblogic-operator-cm` to open up an editor for the operator settings, and then use this editor to add parameter `introspectorJobActiveDeadlineSeconds` with a value of at least 300 seconds (default is 120 seconds).  
+
+TBD Is this a one time cost (first time), or does this happen every time a domain is restarted?
```

Comment:
This is the one time cost.  During update, the import key tells the opss installer to reuse the database.  But I suggest we add the attribute to the helm chart and also fix system test and other classes necessary to make it easier.

Reply:
ok - sounds like this is more work than just changing the README - i'll update the wiki plus add you're comment to the to-do list in your wiki

Reply:
updated - left a simpler TBD in-line and updated our to-do list